src/app/public/modules/angular-tree/angular-tree-node.component.html

```diff
@@ -55,15 +55,18 @@
     <div
       class="node-content-wrapper"
       [allowDragoverStyling]="node.allowDragoverStyling()"
+      [attr.data-node-id]="node.id"
       [attr.tabindex]="tabIndex"
       [treeAllowDrop]="node.allowDrop"
       [treeDrag]="node"
       [treeDragEnabled]="node.allowDrag()"
       (contextmenu)="node.mouseAction('contextMenu', $event)"
       (dblclick)="node.mouseAction('dblClick', $event)"
       (focus)="onFocus()"
+      (keydown.ArrowDown)="onArrowDown($event)"
       (keydown.ArrowLeft)="onArrowLeft($event)"
       (keydown.ArrowRight)="onArrowRight($event)"
+      (keydown.ArrowUp)="onArrowUp($event)"
```

Comment:
Arrow keys in IE 11 use `Up` instead of `ArrowUp`, etc. I'd put in a single event listener and create a switch statement to watch for either key.
```
switch (key) {
  case 'up':
  case 'arrowup':
}
```

Reply:
Good catch. Totally agree. This has been updated.